backend/game/turn_actions/attack.py

```diff
@@ -11,9 +16,13 @@ def get_attackable_players(attacking_player, other_players):
     for other_player in other_players:
         if is_attackable(attacking_player_location, other_player):
             attackable_players.append(other_player)
+    if attacking_player.location == Locations.OUTSIDE and attackable_players.__len__() < 1:
```

Comment:
Refactor into a method to make code readable

Reply:
Also never use `__len__()` like that, just use `len(attackable_players)`

Reply:
And line 19-20 should not be be apart of a function to get attackable players, it muddies the purpose of it

Reply:
Good points. Extracted 19-20 to dice resolution and simplified